api/ccapi/include/layer.h

```diff
@@ -295,6 +297,15 @@ inline std::unique_ptr<Layer> FullyConnected(
   return createLayer(LayerType::LAYER_FC, properties, compute_engine);
 }
 
+/**
+ * @brief Helper function to create SwigluCl layer
+ */
+inline std::unique_ptr<Layer>
+SwigluCl(const std::vector<std::string> &properties = {},
```

Comment:
`SwigluCl` doesn't seem to be consistent with the previous implementation (e.g., fc_layer_cl). Naming with Cl seems only to support cl kernel. What do you think about this?
I think separately exposing layer for cl version for API is not a good idea.
I understood compute_engine parameter is used to hide this.

Reply:
The CPU version of SwiGLU is not yet present for the NNTrainer. We can update this API once both CPU and GPU are available. If required I can change the naming of the layer and remove cl from it for now, what do you say?

Reply:
About the API policy, we may need to listen other reviewers' opinion as well. In my opinion, what about leaving `@todo` in comment to replace the API later ?

Reply:
Added a generalized naming for the swiglu layer in the latest commit, Please have a look and let me know if you think @todo comment is still required.